packages/base-styles/_colors.scss

```diff
@@ -25,6 +25,15 @@ $light-gray-200: #f3f4f5;
 $light-gray-100: #f8f9f9;
 $white: #fff;
 
+// G2 colors.
+$dark-gray-primary: #1e1e1e;
+$medium-gray-text: #757575;
+$light-gray-secondary: #ccc;
+$light-gray-tertiary: #e7e8e9;
+$theme-color: theme(button);
+$blue-medium-focus: #007cba;	// @todo: Currently being used as "spot" color. Needs to be considered in context of themes.
```

Comment:
should we rename this $theme-color-focus and just use SASS functions (tint or shade) to compute it based on `$theme-color`, that way it works  properly no matter the theme color.

Reply:
Mmmmmaybe. As far as is possible at all, we should try and avoid the SASS coloring function as it can never be as good as a manually chosen shade. However if the resting state color is manually picked, it's probably fine if hover and active states are SASS shaded.